.github/renovate.json5

```diff
@@ -13,13 +13,14 @@
     "ignorePaths": [
     ],
 
-    "ignoreDeps": [
-    ],
+    "ignoreDeps": ["avaplatform/avalanchego],
 
     "pinDigests": true,
 
     "enabledManagers": ["helm-values", "helmv3", "regex"],
 
+    "excludeCommitPaths": ["../charts/avalanche/**"],
```

Comment:
excludeCommitPaths is for, after processing an update, excluding potentially changed files from being added to a PR, but doesn't stop Renovate from doing the work up to the point of "not adding changed files to the PR". I guess facing an "empty commit" Renovate won't go trough with actually trying to make a PR with it, but will still have done all the previous processing (and have created the branch)
I think the "enabled: false" below will actually stop it right at the start, with full coverage, so let's just go with that one?